src/sst/elements/memHierarchy/Makefile.am

```diff
@@ -424,6 +424,15 @@ nobase_sst_HEADERS += membackend/ramulatorBackend.h
 AM_CPPFLAGS += $(RAMULATOR_CPPFLAGS) -DHAVE_LIBRAMULATOR
 endif
 
+if HAVE_RAMULATOR2
```

Comment:
You will also need to add all new memH files (sdl, ramulator config, ref file, etc.) to "EXTRA_DIST" in this file.